### DISCUSSION TITLE
Libreelec 11.0

### DISCUSSION THREAD

**Discussion by urlaub33** [CastagnaIT/plugin.video.netflix#1431] [Q&A] [answered]
Hi

After installing Libreelec 11.0 last image from 09/30/22 comes Netflix error message. Kodi log attached.
[kodi.log](https://github.com/CastagnaIT/plugin.video.netflix/files/9686646/kodi.log)

Sorry, this is my first post, I hope I'm in the right place

**Comment by CastagnaIT** [comment, marked as answer]
sorry for the delay but this type of error has been already solved on last version
